### DISCUSSION TITLE
Trailing slash configuration

### DISCUSSION THREAD

**Discussion by rmarscher** [wakujs/waku#873] [Ideas]
I think waku probably needs a trailing slash configuration that build adapters can try to follow when adapting the output for how that platform/runtime handles static file routing since that is platform specific. ChatGPT gave me this summary of the trailingSlash config options in a few meta frameworks:

<table>
  <thead>
    <tr>
      <th>Framework</th>
      <th>Config Option</th>
      <th>Default</th>
      <th>Values</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><strong>Next.js</strong></td>
      <td><code>trailingSlash</code></td>
      <td><code>false</code></td>
      <td><code>true</code>, <code>false</code></td>
    </tr>
    <tr>
      <td><strong>Nuxt.js</strong></td>
      <td><code>router.trailingSlash</code></td>
      <td><code>undefined</code></td>
      <td><code>true</code>, <code>false</code>, <code>undefined</code></td>
    </tr>
    <tr>
      <td><strong>Gatsby</strong></td>
      <td><code>trailingSlash</code></td>
      <td><code>"always"</code></td>
      <td>
        <code>"always"</code>, <code>"never"</code>, <code>"ignore"</code>
      </td>
    </tr>
    <tr>
      <td><strong>SvelteKit</strong></td>
      <td><code>kit.trailingSlash</code></td>
      <td><code>ignore</code></td>
      <td>
        <code>"always"</code>, <code>"never"</code>, <code>"ignore"</code>
      </td>
    </tr>
    <tr>
      <td><strong>Remix</strong></td>
      <td>N/A</td>
      <td>N/A</td>
      <td>Custom middleware or route handling</td>
    </tr>
  </tbody>
</table>

**Comment by dai-shi** [comment]
cc @sandren who has once raised it, but it's postponed.
I need think about at which layer this feature would be added.
